app/components/UI/Stake/Views/StakeInputView/StakeInputView.tsx

```diff
@@ -119,16 +108,27 @@ const StakeInputView = ({ route }: StakeInputViewProps) => {
       return;
     }
 
+    const amount = amountWei.toString();
```

Comment:
Minor: Let's keep wei in the variable name or select another name that is also descriptive so that it's easier to understand at a glance. `amount` is more ambiguous.

```suggestion
    // Or another name of your choosing
    const amountWeiString = amountWei.toString();
```